installer/conf/td-agent-bit.conf

```diff
@@ -23,10 +23,37 @@
     Mem_Buf_Limit 2m
     Path_Key filepath
     Skip_Long_Lines On
+    Ignore_Older 5m
+
+[INPUT]
+    Name tail
+    Tag oms.container.log.telegraf.err.*
+    Path /var/opt/microsoft/docker-cimprov/log/telegraf.log
+    DB /var/opt/microsoft/docker-cimprov/state/telegraf-log-state.db
+    Mem_Buf_Limit 2m
+    Path_Key filepath
+    Skip_Long_Lines On
+    Ignore_Older 5m
+
+[INPUT]
+    Name        tcp
+    Tag oms.container.perf.telegraf.*
+    Listen      0.0.0.0
+    Port        25226
+    Chunk_Size  32
+    Buffer_Size 64
+
+[FILTER]
+    Name grep
+    Match oms.container.log.telegraf.err.*
+    #Regex log /^(?:(?!\[azure_monitor\]: failed to write batch: \[403\] 403 Forbidden).)*$/
 
 [OUTPUT]
     Name                            oms
     EnableTelemetry                 true
     TelemetryPushIntervalSeconds    300
-    Match                           oms.container.log.*
-    AgentVersion                    ciprod03122019
\ No newline at end of file
+    Match                           oms.container.*
+
+#[OUTPUT]
```

Comment:
Should these be removed?

Reply:
This is the one that gets oms agent container logs for telemetry. So we need it.